src/blocks/form/index.js

```diff
@@ -11,6 +13,9 @@ import variations from './variations';
 import { __ } from '@wordpress/i18n';
 import { InnerBlocks } from '@wordpress/block-editor';
 
+// Note: Check that coblocksBlockData is set. So jest tests will pass.
+const successTextDefault = typeof coblocksBlockData === 'undefined' ? __( 'Your message was sent:', 'coblocks' ) : 'coblocksBlockData.form.successText';
```

Comment:
'coblocksBlockData.form.successText' should be coblocksBlockData.form.successText